csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/TrainingTest.cs

```diff
@@ -530,6 +530,90 @@ public void TestSetSeed()
             TrainingUtils.SetSeed(8888);
         }
 
+        [Fact(DisplayName = "TestGetParameter")]
+        public void TestGetParameter()
+        {
+            string checkpointPath = Path.Combine(Directory.GetCurrentDirectory(), "checkpoint.ckpt");
+            using (var cleanUp = new DisposableListTest<IDisposable>())
```

Comment:
>using (var cleanUp = new DisposableListTest<IDisposable>()) [](http://example.com/codeflow?start=12&length=59)

The language version I set for inference API allows inline using clauses, so the disposable list is no longer necessary.

Check what language version your project is using and set it to default.

[https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version)